docs/meshstack.identity-lookup.md

```diff
@@ -166,7 +168,34 @@ let exampleWithSchmaExtension
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
 
-To use a non-standard attribute as
+To use a non-standard attribute as euid
+
+<!--snippet:mesh.meshfed.identitylookup.azure.euid-user-attribute#type-->
+
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Dhall Type-->
+```dhall
+let AzureEuidUserAttribute =
+    {-
+      When adding/inviting a new user to a customer a (custom) attribute property from the users AAD
+      schema can be used to fill in his euid. For a list of user attributes for the AAD user object
+      see:
+
+      https://docs.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0#json-representation
+
+      Attention: This check is only performed on the first attempt when a user is added/invited
+      to a customer. If this check is configured after some users were initially added to a
+      customer their euid's are not udpated.
+
+        userAttributeName:
+            The AAD's user (custom) attribute.
+    -}
+      { userAttributeName : Text }
```

Comment:
t: I really appreciate when documentation comes with examples of "common real world" values because it helps illustrate how I'd actually use it. You can simply add one or multiple `let exampleX : AzureEuidUserAttribute = {...}`within the same source snippet block to include it in the docs

Reply:
Added some examples.